dashboard/data/data.py

```diff
@@ -36,6 +37,41 @@ def has_planning_by(period, client):
     return df[mask]
 
 
+def redenna_by_completed_status(project_name, click_filter=None):
```

Comment:
Hoe heb je getest of de getallen die hieruit rollen ook daadwerkelijk kloppen met wat er in de projectdata staat?